README.md

```diff
@@ -1,11 +1,11 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 # Project Name
-> Porfolio: Desktop Version
+> Porfolio: Mobile/Desktop Version
 This is a Microverse project, week 2 - Module 1 for newly admitted students. It is precisely on building a personal portfolio, desktop version. 
 ## Built With
 - HTML and CSS
 To get a local copy up and running follow these simple example steps:
-1. follow this link - https://github.com/Smart1-hub/Ademola-Portfolio
+1. follow this link - https://smart1-hub.github.io/Project-Portfolio
```

Comment:
- [ ] Kindly document your README properly by providing instructions on how to clone your repo. Your live demo link should not be placed where the repo link is meant to be, so you can create a new section called `LIVE DEMO` and then add your live demo link under it. Something like this 👇 

<img width="287" alt="chrome_feusIY5KR5" src="https://user-images.githubusercontent.com/81468675/161571616-b78a5609-f521-478d-bbec-bc5a450dc480.png">